### ISSUE TITLE
Help File needs to be written

### ISSUE THREAD

**Issue by scotthibbs** [w7sst/MorseRunner#67]
## Summary

Currently our about help menu item will be pointed to our readme file or something similar. Eventually we will want this to pint to a help file that we can create and customize. 

## Motivation

By having our own help file, we can customize support help with various contests as well as new features within our Community Edition. 

## Detailed Description

A text file that describes all the features of our program as well as contests (maybe even when they usually happen with links). 

## Can you help?

Please let us know if you are available to help. (replace '[ ]' with '[x]' to affirm)
- [x] Yes, I'm available to discuss possible solutions and alternatives to this problem.
- [x] Yes, I'd like to review and provide feedback on the proposed solution.
- [x] Yes, I'm available to help test a solution to this problem.

**Comment by scotthibbs** [comment]
I need to write a "First Time?" menu button for the menu bar.

The first item should be "First Time Setup" and should include a quick setup on how the program is setup and used.

Under this should be the name of all our contests in order of the contest dropdown, and if clicked, provide help with each contest and should include blurbs on the contests and exchanges with info on the real dates and links to these contests.

I've assigned myself the texts to write and once done, we can program the menu buttons to be populated.

**Comment by scotthibbs** [comment]
First Time menu item verbiage:

First Time? 

Welcome! This is the first time setup:

1. Select the Contest you wish to operate. 
2. Type the exchange you wish to send. 
3. In the station section replace VE3NEA with your call.
4. Select your CW Speed and Tone
5. Under band conditions you can select any realistic hardships.
6. Activity is the average amount of responses you want per CQ.
	So if no one responds, you might get twice the number the following time.
	This is a pile up trainer after all. - have fun.
7. Select the time limit you want to run the contest.
8. The Run button has a drop down. 
	Single Call - immediately you will get someone trying to contact you. 
	Pile up - You need to call CQ to start. hit enter or F1 

Don't forget to provide feedback at www.hithub.com/w7sst/MorseRunner/

**Comment by w7sst** [comment]
Sounds good.

Don't forget to mention David's @ct7aup new Setup menu items for setting
the min/max received CW speeds.

-Mike

**Comment by w7sst** [comment]
Thank you for writing the first-time user help above. For now, we can drop this into a MessageBox with a simple menu pick.

Concerning a full help file, we will want to learn how Delphi environment supports integrating a Help file. I guess we need an expert who has integrated a Help file into a Delphi Window's application before.

**Comment by scotthibbs** [comment]
True.. And I was reading the readme.txt again and it does work very well as a help file. I need to add my first time text in the configuration section of that file. Before we release a new version I'll have to rewrite the readme to update it. 

For now, let's do the MessageBox with the menu pick so we can close this. 

I'll start another issue for a menu named "Contest info" or something to that affect to give info on each contest.

**Comment by scotthibbs** [comment]
Adding this here for reference: 

I think I understand now - yes, this readme.txt also serves as the Help file. The future readme file will probably be more than a simple text file, perhaps using Windows Help file format, or even a PDF. It would be nice to find a tool for building a help file. We also want a Help file design that will lead to future localization. We want to pick something that be integrated into MRCE to allow F1 key to work (when not in Run Mode). So much to consider. -W7SST
source: https://github.com/w7sst/MorseRunner/pull/161#discussion_r1039028522
